### ISSUE TITLE
Web access

### ISSUE THREAD

**Issue by AlenHodnik** [mletenay/home-assistant-tigo#1]
Can't get past the username/password stage. Is Tigo/$olar always the right user/pass or can be smth else? Does installer of the inverter maybe set this?

**Comment by AlenHodnik** [comment]
Yes. Tried to, but seems passwords in my case dont work. Oh well.

**Comment by chicagoandy** [comment]
If you're getting the 'Done' screen after calling cgi-bin/shell, then you should be able to SSH in using the local IP.

I would double-check the basics like IP address, firewall, network connectivity.

Also doublecheck that you're using this CCA: https://www.tigoenergy.com/product/cloud-connect-advanced

**Comment by mletenay** [comment]
See the post here - https://diysolarforum.com/threads/getting-data-directly-from-a-tigo-tap-is-it-possible.37414/post-1133295
You may have older fw where the user:tigo1 credentials are used and old ssh cipher may need to be enabled.

**Comment by ther3zz** [comment]
> With the last firmware (3.7.7) it seems Tigo changed something. Probably there is already a solution, but they guys don't want to publish it yet: https://www.photovoltaikforum.com/thread/149592-details-protokolle-zugang-auf-tigo-cca/?postID=3829812#post3829812
> 

I just got my system up and running, hopefully the root method is shared soon!
